### PR TITLE
Fix/minor UI tweaks

### PR DESCRIPTION
**Pull request by shawnmcmahon**
- Fix or Feature? 
Fix 

- What changed? 
Fixed some spacing issues between buttons and text 

- Files changed? 
Index.html, styles.css

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -106,7 +112,7 @@ section {
   flex-direction: column;
   margin-bottom: 0px;
   margin-right: 10px;
-  margin-top: 20px;
+  /* margin-top: 20px; */
```
Comment (nathanielmillard) [inline styles.css:115]: Looks like you can go ahead and take this comment out if it isn't needed.

File: styles.css [hunk 2]
```diff
@@ -306,12 +318,10 @@ article {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  outline: none;
```
Comment (nathanielmillard) [inline styles.css:309]: Nice job catching these, I feel like you can end up with a lot of extra css when you are trying to nail down what is usually casuing the bahavior you are seeing.

File: styles.css [hunk 3]
```diff
@@ -394,7 +408,7 @@ section {
   flex-direction: column;
   margin-bottom: 0px;
   margin-right: 10px;
-  margin-top: 20px;
+  /* margin-top: 20px; */
```
Comment (nathanielmillard) [inline styles.css:411]: Same comment as before about commented out code.

**Review by nathanielmillard — COMMENTED** [review]
Over all looking good, I'll take a look around and see are any code reccomendations and not just style notes.

**Comment by nathanielmillard** [comment]
Okay few things to look at that are just reccomendations:
Looking at the pattern started [here](https://github.com/shawnmcmahon/static-comp-2/blob/26ee7658b8be9f0293fa5134eba72bb04808e483/index.html#L14) you might look at actually shifitng your  tags inside their corresponding buttons, as opposed to trying to style them as siblings.
`<button>  Test about button </button>` vs ` <button />

Indenting your `Articles` inside their `A` tags could help with legibility.

If you want your cards populating from left to right but as a whole be centered in the page you could have a `flex-start` attribute on the `background` container and then either and `align-sefl: center` or some sort of property focusing on keeping the background container centered, but then letting the cards contained fill in from lef to right as they flex wrap.

File: styles.css [hunk 4]
```diff
@@ -49,6 +49,12 @@ p {
   margin: 3px;
 }
 
+h1 {
+  display: flex;
+  flex-direction: column;
+  margin: 3px;
+}
```
Comment (BigBike96) [inline styles.css:56]: I like your use of flex and margin. I usually use grid myself

**Review by BigBike96 — COMMENTED** [review]
changes look reasonable

**Pull request merged**